boards/arm/efr32_radio/board.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018 Christian Taedcke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+/* This pin is used to enable the serial port using the board controller */
```

Comment:
do we really need a board.h?

Reply:
I reworked the code and removed board.h.